surveys/2021-annual-survey/questions.md

```diff
@@ -911,16 +911,64 @@ Select one:
 - Yes
 - No [`NEXT`](#excluding-rust-what-is-your-experience-with-other-kinds-of-programming-languages)
 
-### If you are employed, which category best describes the domain in which *you* work?
+### Which category best describes the your current employer's industry?
 
-> TODO - server networking, desktop application, mobile application, web application, embedded, etc.
+Select all that apply:
+
+- Advertising
+- Agricultural
+- Aerospace
+- Automotive
+- Energy
+- Computer hardware
+- Consumer software
+- Business software
+- Finance
+- Food
+- Film or Television
+- Gaming
+- Healthcare
+- News media
```

Comment:
Maybe just 'Media'? Or Traditional or non-social or something? (Which would subsume music and film/tv too, I guess)

Reply:
I've removed news media and combined with film/tv